app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -223,28 +225,78 @@ public void onFailureImpl(DataSource dataSource) {
         }, CallerThreadExecutor.getInstance());
     }
 
+    /**
+     * Calls the set avatar api to set the image url as user's avatar
+     * @param context
+     * @param url
+     * @param username
+     * @param okHttpJsonApiClient
+     * @param compositeDisposable
+     */
+    public static void setAvatarFromImageUrl(Context context, String url, String username,
+        OkHttpJsonApiClient okHttpJsonApiClient, CompositeDisposable compositeDisposable) {
+        showSettingAvatarProgressBar(context);
+
+        try {
+            compositeDisposable.add(okHttpJsonApiClient
+                .setAvatar(username, url)
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                    response -> {
+                        if (response != null && response.getStatus().equals("200")) {
```

Comment:
Is there no Java/Android pre-built constant for "200"? If not it's OK like this :-)